03b_model/src/run_model.py

```diff
@@ -159,33 +272,95 @@ def prep_input_target_data(inputs_xarray, target_xarray,
 
 def write_model_params(out_dir, run_id, inputs, n_epochs_pre,
                        learn_rate_pre, seq_len, hidden_units,
+                       recur_dropout, dropout,
                        train_start_date, train_end_date,
                        val_start_date, val_end_date,
-                       test_start_date, test_end_date):
+                       test_start_date, test_end_date, inc_ante):
+    '''
+    Write model parameters to directory where model results will be saved
+
+    Parameters
+    ----------
+    out_dir : str
+        directory where model results will be saved
+    run_id : str
+        sub-directory within out_dir where model results will be saved
+    inputs : list
+        list of input variables used as predictors
+    n_epochs_pre : int
+        number of epochs to run model
+    learn_rate_pre : float
+        learning rate
+    seq_len : int
+        sequence length
+    hidden_units : int
+        number of hidden units in the model
+    recur_dropout : float
+        fraction of the units to drop from the cell update vector
+    dropout : float
+        fraction of the units to drop from the input
+    train_start_date : str
+        start of training in YYYY-MM-DD format
+    train_end_date : str
+        end of training in YYYY-MM-DD format
+    val_start_date : str
+        start of validation in YYYY-MM-DD format
+    val_end_date : str
+        end of validation in YYYY-MM-DD format
+    test_start_date : str
+        start of testing in YYYY-MM-DD format
+    test_end_date : str
+        end of testing in YYYY-MM-DD format
+    inc_ante : bool
+        flag to determine if antecedant conditions should be explicitly included, if True, 
+        a rolling look back average of the last n days will be included as an input variable, 
+        the number of days and variable are written in the config file
+
+    Returns
+    -------
+    None.
+
+    '''
+    
+    
     #write out model paramters
+    with open("03b_model/model_config.yaml", 'r') as stream:
+        config = yaml.safe_load(stream)
```

Comment:
We should be able to delete this because we read in the config as a global variable in line 23
```suggestion
```

Reply:
I deleted this